gridded_etl_tools/dataset_manager.py

```diff
@@ -120,9 +121,12 @@ def __init__(
             will fail with a warning message.
         use_local_zarr_jsons: bool, optional
             Write out Zarr JSONs created via Kerchunk to the local file system. For use with remotely kerchunked
-            datasets.
+            datasets. Defaults to True to speed up post-parse quality checks.
```

Comment:
This param actually defaults to False which also does seem like the reasonable default, but if it makes more sense what it could do is default to None and then unless it is manually set by the caller, it gets set to True when skip_post_parse_qc is False or False otherwise

Reply:
It’s actually not a good idea to set it to